File: CS/GridDemo.Wpf/ModuleResources/PrintOptionsViewModel.cs
using System;
using DevExpress.Mvvm.DataAnnotations;
using DevExpress.Xpf.Printing;

namespace GridDemo {
    [POCOViewModel]
    public class PrintOptionsViewModel : PrintViewModelBase {
        protected PrintOptionsViewModel() { }
        public virtual bool UseCustomPrintStyles { get; set; }

        public override void OnLoaded(IPrintableControl printableControl) {
            base.OnLoaded(printableControl);
            ShowPreviewInNewTab();
            UseCustomPrintStyles = true;
            ShowPreviewInNewTab();
            UseCustomPrintStyles = false;
        }
        protected override string GetTitle() {
            return (UseCustomPrintStyles ? "Custom" : "Default") + " Style Print Preview";
        }

    }
}

File: VB/GridDemo.Wpf/ModuleResources/PrintOptionsViewModel.vb
Imports System
Imports DevExpress.Mvvm.DataAnnotations
Imports DevExpress.Xpf.Printing

Namespace GridDemo
    <POCOViewModel>
    Public Class PrintOptionsViewModel
        Inherits PrintViewModelBase

        Protected Sub New()
        End Sub
        Public Overridable Property UseCustomPrintStyles() As Boolean

        Public Overrides Sub OnLoaded(ByVal printableControl As IPrintableControl)
            MyBase.OnLoaded(printableControl)
            ShowPreviewInNewTab()
            UseCustomPrintStyles = True
            ShowPreviewInNewTab()
            UseCustomPrintStyles = False
        End Sub
        Protected Overrides Function GetTitle() As String
            Return (If(UseCustomPrintStyles, "Custom", "Default")) & " Style Print Preview"
        End Function

    End Class
End Namespace
